client/ting.jsx

```diff
@@ -127,6 +127,8 @@ const Ting = React.createClass({
         this._socket.emit('login', intendedUsername);
     },
     render() {
+        var audio = new Audio('static/sounds/message_sound.mp3');
+
```

Comment:
This recreates a new audio every time the render function is called. Probably not what you want. React is free to call this function as much as it likes.